packages/test/test-version-utils/src/versionUtils.ts

```diff
@@ -212,16 +212,24 @@ export async function ensureInstalled(
 			};
 			// Install the packages
 			await new Promise<void>((resolve, reject) =>
-				exec(`npm init --yes`, options, (error, stdout, stderr) => {
+				// Added --verbose to try to troubleshoot AB#6195.
+				// We should probably remove it if when find the root cause and fix for that.
+				exec(`npm init --yes --verbose`, options, (error, stdout, stderr) => {
 					if (error) {
-						reject(new Error(`Failed to initialize install directory ${modulePath}`));
+						reject(
+							new Error(
+								`Failed to initialize install directory ${modulePath}\n${stderr}`,
```

Comment:
Added `stderr` from the executed command to the output, like the block below does.